tests/test_tokens_prices.py

```diff
@@ -0,0 +1,65 @@
+import setup
+import pytest
+import timeit
+import time
+
+from datetime import datetime
+from sqs_service import *
+from coingecko_service import *
+from conftest import SERVICE_MAP
+from conftest import SERVICE_COINGECKO
+from constants import *
+
+class TestTokensPrices:
```

Comment:
I would like to suggest the following test cases:
- All valid listed tokens as defined by [this function](https://github.com/osmosis-labs/sqs/blob/3c66c5f7fe24c223fd73ff3cd9c0b1adefc69ff6/tests/setup.py#L263)
  * Track all the prices that we do not support and print them
  * Assert that the number of unsupported tokens is below some pre-configured threshold. I think that 30 is a good starting value
- Multiple base denoms requested in a single query.

Additionally, let's also add support for the SQS CoinGecko integration price test (can be a separate PR) - just noting so that we don't forget

Reply:
Test cases added as suggested and separate PR created as it depends on coingecko integration PR merge.